### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #18: Write CONTRIBUTING.MD
- guide to git workflow
- add badges linking to issues/prs/etc
- explain travis setup & PR merge workflow
- explain stylechecker

### PR DESCRIPTION
**Pull request by julian-zucker**
also deleted some TODOs that already exist in our issues.

closes #18

### CODE REVIEW

**Comment by julian-zucker** [comment]
I wrote "closing #27", "not closes #27", so it didn't automatically link this PR to closing that issue. I'm going to hope that saying closes #27 here will do the trick

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#### ScalaStyle
+We use [`scalastyle`](http://www.scalastyle.org) as our style checker. You can find the configuration at [project/scalastyle_config.xml](project/scalastyle_config.xml). This style configuration is liable to change, and we are open to suggestions – but make sure that any code you write passes the style configuration you have submitted. You can manually run the style checker with `mvn scalastyle:check`, which will print the filename and line number of everyu violation. You can also set up a pre-commit hook to run the style checker, as described [in the scalastyle docs](http://www.scalastyle.org/git-pre-commit-hook.html). Scalastyle is run on every pull request.
```
Comment (drassaby) [inline CONTRIBUTING.md]: Typo: everyu -> every

Comment (drassaby) [inline CONTRIBUTING.md]: Liable to changes -> may change over time

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+### Contributors
+
+Evvo is written by:
+
+* Julian Zucker <julian.zucker@gmail.com>
```
Comment (drassaby) [inline CONTRIBUTORS.md:5]: Can you add me too lool

**Pull request merged**